packages/flutter_tools/lib/src/project_validator.dart

```diff
@@ -109,3 +112,73 @@ class GeneralInfoProjectValidator extends ProjectValidator{
   @override
   String get title => 'General Info';
 }
+
+class PubDependenciesProjectValidator extends ProjectValidator {
+  PubDependenciesProjectValidator(this._processManager);
```

Comment:
```suggestion
  const PubDependenciesProjectValidator(this._processManager);
```

Reply:
I couldn't implement this because of the abstract class

Reply:
Can you make the super also const?

Reply:
you'll have to make `title` final, but i don't think that should be a problem